shared_model/validators/protobuf/proto_transaction_validator.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "validators/protobuf/proto_transaction_validator.hpp"
+
+#include "validators/validators_common.hpp"
+
+namespace shared_model {
+  namespace validation {
+
+    void validateHexField(const std::string &public_key,
```

Comment:
The function name does not match the message of validation failure reason.
```suggestion
    void validatePublicKeyField(const std::string &public_key,
```